libs/core/src/components/pds-button/docs/pds-button.mdx

```diff
@@ -191,6 +191,56 @@ The button submits the form data to the server.
 
 The button resets all the controls to their initial values. [As noted on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type), this behavior tends to annoy users.
 
+
+## Styling with CSS Parts
+
+The `pds-button` component provides CSS parts that allow for targeted customization. These parts should be used sparingly and only when the built-in props do not provide the necessary styling flexibility.
+
```

Comment:
I like that we're adding this but considering other components also have CSS Parts, should we consider making a separate page to talk about them in general?

Reply:
@ju-Skinner, how do you feel about this instead of per component examples? Let me know. I can roll it back here and create a ticket for a new page or something when time permits.

Reply:
Per a conversation between the three of us, we have decided to remove the documentation examples for now.